com.giuspen.cherrytree.json

```diff
@@ -0,0 +1,128 @@
+{
+  "app-id": "com.giuspen.cherrytree",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cherrytree",
+  "rename-desktop-file": "cherrytree.desktop",
+  "rename-icon": "cherrytree",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host"
+  ],
+  "cleanup": [
+    "/include",
+    "/share/bash-completion",
+    "/lib/pkgconfig/",
+    "/lib/*.a",
+    "/lib/*.la",
+    "/lib/cairo/*.a",
+    "/lib/cairo/*.la",
+    "/share/doc/",
+    "/man/"
+  ],
+  "modules": [
+    "shared-modules/intltool/intltool-0.51.json",
+    {
+      "name": "cairo",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.cairographics.org/releases/cairo-1.16.0.tar.xz",
+          "sha256": "5e7b29b3f113ef870d1e3ecf8adf21f923396401604bda16d44be45e66052331"
+        }
+      ]
+    },
+    "shared-modules/gtk2/gtk2.json",
+    {
+      "name": "gtksourceview",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/gtksourceview/2.10/gtksourceview-2.10.5.tar.bz2",
+          "sha256": "c585773743b1df8a04b1be7f7d90eecdf22681490d6810be54c81a7ae152191e"
+        }
+      ]
+    },
+    "shared-modules/python2.7/python-2.7.json",
+    {
+      "name": "pycairo",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dpython=python"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/pygobject/pycairo/archive/v1.18.1.zip",
+          "sha256": "7340f8c7e164718bcc5de23d4bcff39ac81994a070edefdcdbcd1e31c9502754"
+        }
+      ]
+    },
+    {
+      "name": "pygobject",
+      "config-opts": [
+        "--disable-introspection"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/pygobject/2.28/pygobject-2.28.6.tar.xz",
+          "sha256": "fb8a1d4f665130a125011659bd347c7339c944232163dbb9a34fd0686577adb8"
+        }
+      ]
+    },
+    {
+      "name": "pygtk",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2",
+          "sha256": "cd1c1ea265bd63ff669e92a2d3c2a88eb26bcd9e5363e0f82c896e649f206912"
+        }
```

Comment:
You need to add a new source to pygtk as below
```
                {
                    "type": "shell",
                    "commands": [
                        "cp -p /usr/share/automake-*/config.{sub,guess} ."
                    ]
                }
```

Reply:
This should fix the build in x86_64 at least for now

Reply:
Thanks!